### DISCUSSION TITLE
Unable to access android/data on android 11

### DISCUSSION THREAD

**Discussion by test2a** [PaulWoitaschek/Voice#1333] [Ideas]
So this seems to be android 11 thing and there are either two things than can be done. Just like material files, it has a separate external storage option that let's you access the folder or allow users to pass file picker to external apps like material Fies which can then access those files. 

I suppose voice needs to enable support for those folders anyways.

**Comment by yhack** [comment]
I just upgraded to Android 11 and now face this issue. In Android 11 files in Android/data are protected so only the apps that created them can access them.

It is possible for apps to gain access to all of Android/data using another method. Some that have done it are listed here https://www.reddit.com/r/Android/comments/j3zgmm/managing_files_in_the_androiddata_folder_on/.

How to achieve this has an answer on stack overflow https://stackoverflow.com/questions/65967690/how-do-some-apps-reach-the-contents-of-android-sub-folders-on-android.

It would be great to have this working in Voice.

**Comment by PaulWoitaschek** [comment]
Why would you like to access that folder?

**Comment by test2a** [comment]
@PaulWoitaschek i have an app that downloads mp3 to its data folder. so instead of using that app, i have for like last 3 years used voice. Now, the last few months i got a new phone and android 11. i am forced to manually copy each item from the data folder to one in documents/downloads where the voice picks them up. 

earlier it was painless. Now, not so much

**Comment by yhack** [comment]
OverDrive saves books there. Specifically on the SD card at `/storage/0000-0000/Android/data/com.overdrive.android.mediaconsole/files`.

**Comment by test2a** [comment]
@yhack i can't accept or deny i was intending the same ;-)

**Comment by dschaeffer** [comment]
Another person here who would be happy for this feature.
